lib/pure/collections/intsets.nim

```diff
@@ -42,13 +52,17 @@ type
     bits: array[0..IntsPerTrunk - 1, BitScalar] # a bit vector
 
   TrunkSeq = seq[PTrunk]
-  IntSet* = object       ## An efficient set of `int` implemented as a sparse bit set.
+
+  ## A must be an ordinal equivalent type: int | enum | distinct of an int
+  OrdSet*[A] = object    ## An efficient set of `int` implemented as a sparse bit set.
```

Comment:
shouldn't it be: `OrdSet*[A: Ordinal]` or something similar?

Reply:
I hadn't been able to make [A: Ordinal] work and now I see that apparently you cannot have both the type and the procs in the api constrained to ordinal. Nevertheless unconstraining the Api and making the type constrained works and it's better for the type safety. Changed that.